### PR TITLE
move function answerLinks to utils folder

### PR DESCRIPTION
**Pull request by Andryushik**

### CODE REVIEW

File: src/utils/answerLinksUtil.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { LINKS_DIV } from '../constants.js';
+import { createLinkElement } from '../views/linkView.js';
+
+export const answerLinks = (currentQuestion) => {
```
Comment (mnvacym) [inline src/utils/answerLinksUtil.js:4]: I think this function's responsibility is to append links. Should we rename the function as `appendLinks`? What do you think?

Comment (Andryushik) [reply src/utils/answerLinksUtil.js:4]: sure

**Review by ysfelik1 — APPROVED** [review]
okay well done

File: src/pages/questionPage.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
   ANSWERS_LIST_ID,
-  LINKS_DIV,
   NEXT_QUESTION_BUTTON_ID,
   USER_INTERFACE_ID,
 } from '../constants.js';
 import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
-import { createLinkElement } from '../views/linkView.js';
+import { appendLinks } from '../utils/answerLinksUtil.js';
```
Comment (mnvacym) [inline src/pages/questionPage.js:9]: Please also update the file name

Comment (Andryushik) [reply src/pages/questionPage.js]: done

**Comment by ysfelik1** [comment]
branch merged.

**Pull request merged**